src/main/java/net/dv8tion/jda/internal/entities/EntityBuilder.java

```diff
@@ -2099,9 +2099,23 @@ public ApplicationInfo createApplicationInfo(DataObject object)
         final boolean isBotPublic = object.getBoolean("bot_public");
         final User owner = createUser(object.getObject("owner"));
         final ApplicationTeam team = !object.isNull("team") ? createApplicationTeam(object.getObject("team")) : null;
+        final String customAuthUrl = object.getString("custom_install_url", null);
+        final List<String> tags = object.optArray("tags").orElseGet(DataArray::empty)
+                    .stream(DataArray::getString)
+                    .collect(Collectors.toList());
+
+        final long defaultAuthUrlPerms = object.optObject("install_params")
+                    .map(o -> o.getLong("permissions"))
+                    .orElse(0L);
+
+        final List<String> defaultAuthUrlScopes = object.optObject("install_params")
```

Comment:
Put `object.optObject("install_params")` into a variable so it doesn't have to allocate twice